projects/storefrontapp-e2e-cypress/cypress/e2e/b2b/regression/asm/customer360-promotion.e2e.cy.ts

```diff
@@ -111,7 +111,7 @@ context('Assisted Service Module', () => {
     it('should be able to sent customer coupon for customer coupon (CXSPA-3945)', () => {
       interceptPost(
         'claim_customer_coupon',
-        '/users/*/customercoupons/*/claim?*'
+        '/users/*/customercoupons/*/claim?*' //TODO use '/users/*/customercoupons/claim?*' instead when enable 'enableClaimCustomerCouponWithCodeInRequestBody' which is available since Commerce 2211.28
```

Comment:
hi @rmch91 , not sure whether you are enjoying the holiday, hope not bother you. As I already disable the toggle to avoid break CI because of  deployed commerce version issue. And I skip new added e2e test, but for the existed test cases, I left comments like this.  Because when enable 'enableClaimCustomerCouponWithCodeInRequestBody', the case will failed because it need check new occ  endpoints calling , so we need update the case here after enable 'enableClaimCustomerCouponWithCodeInRequestBody', do you think the left comments is enough for future update? or is there other steps for it? creating jira tickets to follow it?
Thanks.